### ISSUE TITLE
No module pax.plugins.interaction_processing.S1AreaFraktionTopProbability

### ISSUE THREAD

**Issue by XeBoris** [XENON1T/pax#675]
Dear all,

While running cax in the slow control screen session I see this error message (taken out from cax.log on xe1t-datamanager):

```
cax_v5.2.1 - 2018-02-16 11:33:03,188 [ERROR] No module named 'pax.plugins.interaction_processing.S1AreaFractionTopProbability'
Traceback (most recent call last):
  File "/home/xe1tdatamanager/cax_fl_test/cax/main.py", line 147, in main
    task.go(args.run)
  File "/home/xe1tdatamanager/cax_fl_test/cax/task.py", line 65, in go
    self.each_run()
  File "/home/xe1tdatamanager/cax_fl_test/cax/tasks/corrections.py", line 78, in each_run
    {'$set': {self.key: self.evaluate(),
  File "/home/xe1tdatamanager/cax_fl_test/cax/tasks/corrections.py", line 138, in evaluate
    gains = self.get_gains(timestamp)
  File "/home/xe1tdatamanager/cax_fl_test/cax/tasks/corrections.py", line 156, in get_gains
    hax.init(pax_version_policy='loose', main_data_paths=[])
  File "/home/SHARED/anaconda3/envs/pax_head/lib/python3.4/site-packages/hax-2.4.0-py3.4.egg/hax/__init__.py", line 71, in init
    minitrees.update_treemakers()
  File "/home/SHARED/anaconda3/envs/pax_head/lib/python3.4/site-packages/hax-2.4.0-py3.4.egg/hax/minitrees.py", line 147, in update_treemakers
    __import__('hax.treemakers.%s' % module_name, globals=globals())
  File "/home/SHARED/anaconda3/envs/pax_head/lib/ROOT.py", line 301, in _importhook
    return _orig_ihook( name, *args, **kwds )
  File "/home/SHARED/anaconda3/envs/pax_head/lib/python3.4/site-packages/hax-2.4.0-py3.4.egg/hax/treemakers/posrec.py", line 10, in <module>
    from pax.plugins.interaction_processing.S1AreaFractionTopProbability import s1_area_fraction_top_probability
  File "/home/SHARED/anaconda3/envs/pax_head/lib/ROOT.py", line 301, in _importhook
    return _orig_ihook( name, *args, **kwds )
ImportError: No module named 'pax.plugins.interaction_processing.S1AreaFractionTopProbability'
```

Furthermore I can also not run ruciax verifier anymore for this error:
```
    import hax
  File "/cvmfs/xenon.opensciencegrid.org/releases/anaconda/2.4/envs/pax_head/lib/python3.4/site-packages/hax-2.4.0-py3.4.egg/hax/__init__.py", line 7, in <module>
    from . import misc, minitrees, paxroot, pmt_plot, raw_data, runs, utils, treemakers, data_extractor, slow_control, \
  File "/cvmfs/xenon.opensciencegrid.org/releases/anaconda/2.4/envs/pax_head/lib/python3.4/site-packages/hax-2.4.0-py3.4.egg/hax/raw_data.py", line 12, in <module>
    from pax import core
  File "/cvmfs/xenon.opensciencegrid.org/releases/anaconda/2.4/envs/pax_head/lib/ROOT.py", line 301, in _importhook
    return _orig_ihook( name, *args, **kwds )
  File "/cvmfs/xenon.opensciencegrid.org/releases/anaconda/2.4/envs/pax_head/lib/ROOT.py", line 301, in _importhook
    return _orig_ihook( name, *args, **kwds )
  File "/cvmfs/xenon.opensciencegrid.org/releases/anaconda/2.4/envs/pax_head/lib/python3.4/site-packages/pax-6.8.3-py3.4.egg/pax/core.py", line 16, in <module>
    from pax import simulation, utils
  File "/cvmfs/xenon.opensciencegrid.org/releases/anaconda/2.4/envs/pax_head/lib/ROOT.py", line 301, in _importhook
    return _orig_ihook( name, *args, **kwds )
  File "/cvmfs/xenon.opensciencegrid.org/releases/anaconda/2.4/envs/pax_head/lib/ROOT.py", line 301, in _importhook
    return _orig_ihook( name, *args, **kwds )
  File "/cvmfs/xenon.opensciencegrid.org/releases/anaconda/2.4/envs/pax_head/lib/python3.4/site-packages/pax-6.8.3-py3.4.egg/pax/simulation.py", line 21, in <module>
    from pax.InterpolatingMap import InterpolatingMap
  File "/cvmfs/xenon.opensciencegrid.org/releases/anaconda/2.4/envs/pax_head/lib/ROOT.py", line 301, in _importhook
    return _orig_ihook( name, *args, **kwds )
  File "/cvmfs/xenon.opensciencegrid.org/releases/anaconda/2.4/envs/pax_head/lib/python3.4/site-packages/pax-6.8.3-py3.4.egg/pax/InterpolatingMap.py", line 14, in <module>
    class InterpolateAndExtrapolate(object):
  File "/cvmfs/xenon.opensciencegrid.org/releases/anaconda/2.4/envs/pax_head/lib/python3.4/site-packages/pax-6.8.3-py3.4.egg/pax/InterpolatingMap.py", line 38, in InterpolateAndExtrapolate
    v__call__ = np.vectorize(__call__, signature = '(),(i)->()')
TypeError: __init__() got an unexpected keyword argument 'signature'
```

It maps also in direction that something is missing in pax after an update.
Did something changed in the modules recently?

Thank you

**Comment by feigaodm** [comment]
second error is related to PR #672 so I guess @zhut19 should be able to tell what's the problem and how to fix it.

**Comment by JelleAalbers** [comment]
I also hit the second error on the event builder, see #674 . I solved it by not using the latest pax, alternatively you can install a later numpy version via pip.

**Comment by feigaodm** [comment]
Thanks @JelleAalbers, I think @XeBoris can use pax_v6.9.0 instead before @zhut19 fix the second problem. 

Not sure if we will encounter the first problem there, I had use the s1_area_fraction_top_probability function without problem, any idea @pdeperio ?

**Comment by pdeperio** [comment]
Looks like ```xe1t-datamanager``` was removed from the ```Analysis sites``` server group on [DeployHQ](https://xenon1t.deployhq.com/projects/pax/servers). I'm not sure when or why, but I guess the pax installation hasn't been getting updated.

**Comment by JelleAalbers** [comment]
This was solved by #676 (and eventually #722)
